package.json

```diff
@@ -55,6 +55,7 @@
     }
   },
   "scripts": {
+    "prepare": "json2ts ./src/helper/config.schema.json > ./src/app/config/config.model.ts; node -e 'const {writeDefaultConfig} = await import(`./src/helper/config.js`); writeDefaultConfig()'",
     "build:prod": "ng build --configuration production",
```

Comment:
Probably best to have the prod build scripts include a config regen just to be safe? Also maybe have the CI checks also verify it's been updated?

Reply:
The files that are generated are ignored by Git, so the CI does not need to verify they're updated since they're local only.

Regenerating during development, however, sounds like a good idea!